pkg/project/v2/project_loader.go

```diff
@@ -182,28 +182,24 @@ func loadProject(fs afero.Fs, context ProjectLoaderContext, projectDefinition ma
 	log.Debug("Loading project `%s` (%s)...", projectDefinition.Name, projectDefinition.Path)
 
 	configs, errors := loadConfigsOfProject(fs, context, projectDefinition, environments)
+	errors = append(errors, findDuplicatedConfigIdentifiers(configs)...)
+	errors = append(errors, checkKeyUserActionScope(configs)...)
 
-	if d := findDuplicatedConfigIdentifiers(configs); d != nil {
-		for _, c := range d {
-			errors = append(errors, newDuplicateConfigIdentifierError(c))
-		}
-	}
-
-	for _, c := range configs {
-		// The scope parameter of a key user actions web configuration needs to be a reference to another application-web config
-		// The reference parameter makes sure that rely on the fact that kua web configs are loaded/deployed within the same
-		// sub graph (independent component) later on as
-		if c.Coordinate.Type == api.KeyUserActionsWeb {
-			if _, ok := c.Parameters[config.ScopeParameter].(*ref.ReferenceParameter); !ok {
-				errors = append(errors, fmt.Errorf("scope parameter of config of type '%s' with ID '%s' needs to be a reference "+
-					"parameter to another web-application config", api.KeyUserActionsWeb, c.Coordinate.ConfigId))
-			}
-		}
-	}
-	if errors != nil {
+	if len(errors) > 0 {
 		return Project{}, errors
 	}
 
+	insertFixedParameters(configs)
+
+	return Project{
+		Id:           projectDefinition.Name,
+		GroupId:      projectDefinition.Group,
+		Configs:      toConfigMap(configs),
+		Dependencies: toDependenciesMap(projectDefinition.Name, configs),
+	}, nil
+}
+
+func insertFixedParameters(configs []config.Config) {
```

Comment:
I assume this is named with the intention, that it COULD do more things than just adding the one fixed reference for network zones in the future? 
IMO it can be changed to that if it ever happens - for now it'd be nicer to read with a clear name and by moving the documentation of WHY from comments inside the function to one on the function itself. 
Also a note that it modifies the configs it's given might not hurt